### ISSUE TITLE
Cookie expiry leaves UI in an unusable state

### ISSUE THREAD

**Issue by MattNotarangelo** [flipt-io/flipt#1334]
### Describe the Bug

Leaving the session active for >24 hours (default cookie expiration time) causes the session cookie to expire, but the UI doesn't allow any further actions since the user isn't currently authenticated. Attempting to logout doesn't work since this also requires authentication, and refreshing/reloading the page doesn't help. Clearing all cookies/localstorage data forces a redirect to the login page which allows the user to login again.

The issue seems to occur since the logged in state is checked by parsing the localstorage data, but this doesn't expire. This leaves the UI in a state where the localstorage data exists, but the cookie doesn't

### Version Info

Run `flipt --version` and paste the output here:

```
Version: 1.18.1 
Commit: 3effb95f5a3022a153eab4ed337632d450771ecd 
Build Date: 2023-02-02T14:35:03Z 
Go Version: go1.18.10
```

### To Reproduce

1. Log in with OIDC provider
2. Wait for cookie expiry (or delete cookie, or change expiry manually)

### Expected Behaviour

The user should be redirected to the login page

### Additional Context

Config
```
# local.yml

cors:
  enabled: true
  allowed_origins: ["*"]

authentication:
  required: true
  session:
    secure: true
    domain: "http://localhost:8080"
  methods:
    oidc:
      enabled: true
      providers:
        google:
          issuer_url: "https://accounts.google.com"
          client_id: ...
          client_secret: ...
          redirect_address: "http://localhost:8080"
          scopes:
            - email
            - profile
```

OS: Mac OSX 13.1
Browser: Chrome Version 109.0.5414.119 (Official Build) (arm64)

**Comment by GeorgeMac** [comment]
Thanks for the detailed issue @MattNotarangelo 🙏 

I've started to dig in a little. I have an experiment to improve the cookie-clearing mechanics, though I suspect it is not the entire story. I think some digging in the UI repository will be needed next. Will report back.

**Comment by GeorgeMac** [comment]
FYI @MattNotarangelo this will be fixed in 1.18.2

We're just preparing the release now: https://github.com/flipt-io/flipt/pull/1338

**Comment by markphelps** [comment]
@MattNotarangelo this should be fixed now in [v1.18.2](https://github.com/flipt-io/flipt/releases/tag/v1.18.2) as @GeorgeMac mentioned. Please give it a try when you have a chance and let us know if the issue still persists

**Comment by MattNotarangelo** [comment]
Thanks @GeorgeMac @markphelps, the issue seems to be fixed!
